Misc/NEWS.d/next/Core and Builtins/2019-10-20-00-36-18.bpo-38525.Vty1cA.rst

```diff
@@ -0,0 +1,2 @@
+Fix a segmentation fault when using reverse iterators of empty ``dict``
+objects that were created as empty ``dict`` literals. Patch by Dong-hee Na.
```

Comment:
It is implementation detail and not correct (e.g. `d.clear()` makes key-sharing empty dict too).
I think this is enough:

```
Fix a segmentation fault when using reverse iterators of empty dict objects.
```

Reply:
@methane Thanks I've updated it :)